runtime/queue.c

```diff
@@ -775,16 +775,20 @@ qqueueTryLoadPersistedInfo(qqueue_t *pThis)
 	DEFiRet;
 	strm_t *psQIF = NULL;
 	struct stat stat_buf;
+	int errNbr;
 
 	ISOBJ_TYPE_assert(pThis, qqueue);
 
 	/* check if the file exists */
 	if(stat((char*) pThis->pszQIFNam, &stat_buf) == -1) {
-		if(errno == ENOENT) {
+		errNbr = errno;
+		if(errNbr == ENOENT) {
+			LogError(0, RS_RET_IO_ERROR, "queue: clean startup, no .qi file found");
 			DBGOPRINT((obj_t*) pThis, "clean startup, no .qi file found\n");
 			ABORT_FINALIZE(RS_RET_FILE_NOT_FOUND);
 		} else {
-			DBGOPRINT((obj_t*) pThis, "error %d trying to access .qi file\n", errno);
+			LogError(0, RS_RET_IO_ERROR, "queue: error %d could not access .qi file", errNbr);
```

Comment:
errNbr should be given to LogError, so that it can generate a proper error message including the actual OS error.